### ISSUE TITLE
Add explanation to contributions block on journey dashboard

### ISSUE THREAD

**Issue by SimoneZaza** [alkem-io/client-web#4004]
## Description

As a member of private Subspaces, I can see the activity in them on the Space dashboard. It is unclear to me that others, who are not a member, cannot see this activity. This worries me since the things I share in that private Subspace are actually private. 
>> Add help icon in recent contributions block **next to title** explaining what is displayed. (screenshot shows icon in the wrong place, since here the switch is located nowadays)
![image.png](https://images.zenhubusercontent.com/6273938fb2a7cf60e23df935/f32829cb-2c95-435c-a3e0-e4dae152c5e8)

## Acceptance criteria

- [ ] Info icon top right of Contributions block
- [ ] Tooltip with text (translatable): 

> The activity shown in this section is based on the content that you have access to. Contributions in private Subspaces are only visible for members of those Subspaces.

- [ ] Above in recent contributions blocks on Space and Challenge levels

**Comment by hero101** [comment]
This activity is not based on membership. All members of the Space are able to see the entries.
@techsmyth

**Comment by SimoneZaza** [comment]
@hero101 but not the entries of the private challenges, right?

**Comment by hero101** [comment]
@SimoneZaza Yes. I have verified that a user not a member of a private challenge does not see its activity.
Non-members can see the activity of public challenges.
